tests/AllenCahn/input

```diff
@@ -9,6 +9,11 @@
 #@  dim    = 2
 #@  check  = false
 #@  args   = stop_time=1.0
+#@
+#@  [2D-random.ic-test]
+#@  dim    = 2
+#@  check  = true
+#@  args   = type = random
```

Comment:
@mongole584 I ran this test and it doesn't seem to be actually using the new IC. Instead it is running the same test as before. Can you please investigate?